_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -60,7 +59,7 @@
         "acquia/lightning": "~3.2.6",
         "ckeditor/indentblock": "4.11.4",
         "drupal/admin_toolbar": "1.26",
-        "drupal/assembly": "1.x-dev#94645cf6617d5f375ba2cc3c060df13c88e75d8b",
+        "drupal/assembly": "1.x-dev",
```

Comment:
@jordanpagewhite I assume this leaves the potential for this dependency to change between builds. Can we lock it to a commit that you're happy with please?

Reply:
@robpblake I resolved this in the commit I just pushed. Thanks!